### PR TITLE
Run3-hcx328 Replace cout with LogVerbatim and update the cfg's in Geometry/HcalCommonData

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Replace cout with LogVerbatim and update the cfg's in Geometry/HcalCommonData

#### PR validation:

Use the runTheMatrix test workflows and cfg's in the test directory

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
In the comparison test workflow 10042.0 ZMM 2017 had several unexpected differences. How can any change in this PR cause those differences?

File: Geometry/HcalCommonData/src/HcalLayerDepthMap.cc [hunk 1]
```diff
@@ -117,8 +121,8 @@ int HcalLayerDepthMap::getDepth(
       depth = itr->second;
   }
 #ifdef EDM_ML_DEBUG
-  std::cout << "getDepth::Input " << subdet << ":" << ieta << ":" << iphi << ":" << zside << ":" << layer << " Output "
-            << depth << std::endl;
+  edm::LogVerbatim("HCalGeom") << "getDepth::Input " << subdet << ":" << ieta << ":" << iphi << ":" << zside << ":"
```
Comment (cvuosalo) [inline Geometry/HcalCommonData/src/HcalLayerDepthMap.cc:124]:
```suggestion
  edm::LogVerbatim("HCalGeom") << "Debug info -- getDepth::Input " << subdet << ":" << ieta << ":" << iphi << ":" << zside << ":"
```

File: Geometry/HcalCommonData/src/HcalLayerDepthMap.cc [hunk 2]
```diff
@@ -128,31 +132,31 @@ int HcalLayerDepthMap::getDepth16(const int subdet, const int iphi, const int zs
   if (isValid(subdet, iphi, zside))
     depth = dep16C_;
 #ifdef EDM_ML_DEBUG
-  std::cout << "getDepth16::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth << std::endl;
+  edm::LogVerbatim("HCalGeom") << "getDepth16::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth;
```
Comment (cvuosalo) [inline Geometry/HcalCommonData/src/HcalLayerDepthMap.cc:135]:
```suggestion
  edm::LogVerbatim("HCalGeom") << "Debug info -- getDepth16::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth;
```

File: Geometry/HcalCommonData/src/HcalLayerDepthMap.cc [hunk 3]
```diff
@@ -128,31 +132,31 @@ int HcalLayerDepthMap::getDepth16(const int subdet, const int iphi, const int zs
...
 #endif
   return depth;
 }
 
 int HcalLayerDepthMap::getDepthMin(const int subdet, const int iphi, const int zside) const {
   int depth = (isValid(subdet, iphi, zside)) ? depthMin_ : -1;
 #ifdef EDM_ML_DEBUG
-  std::cout << "getDepthMin::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth << std::endl;
+  edm::LogVerbatim("HCalGeom") << "getDepthMin::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth;
```
Comment (cvuosalo) [inline Geometry/HcalCommonData/src/HcalLayerDepthMap.cc:143]:
```suggestion
  edm::LogVerbatim("HCalGeom") << "Debug info -- getDepthMin::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth;
```

File: Geometry/HcalCommonData/src/HcalLayerDepthMap.cc [hunk 4]
```diff
@@ -128,31 +132,31 @@ int HcalLayerDepthMap::getDepth16(const int subdet, const int iphi, const int zs
...
 #endif
   return depth;
 }
 
 int HcalLayerDepthMap::getDepthMax(const int subdet, const int iphi, const int zside) const {
   int depth = (isValid(subdet, iphi, zside)) ? depthMax_ : -1;
 #ifdef EDM_ML_DEBUG
-  std::cout << "getDepthMax::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth << std::endl;
+  edm::LogVerbatim("HCalGeom") << "getDepthMax::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth;
```
Comment (cvuosalo) [inline Geometry/HcalCommonData/src/HcalLayerDepthMap.cc:151]:
```suggestion
  edm::LogVerbatim("HCalGeom") << "Debug info -- getDepthMax::Input " << subdet << ":" << iphi << ":" << zside << " Output " << depth;
```

**Comment by cvuosalo** [comment]
To explain my suggestions above:

As I mentioned in issue #34294, some of the debug messages from `HcalLayerDepthMap.cc` can be needlessly alarming when the `depth` is reported as -1. The message looks like an error to the non-expert. To make more clear that the message is just debugging information and not an error, I am suggesting that explanatory text be added to each message, as shown above.

**Comment by bsunanda** [comment]
@cmsbuild Please check

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
Comparison differences in reco quantities for workflow 10042.0 have appeared again. These differences don't appear in recent tests in other PRs, but I don't see any change in this PR that could cause these differences.
Does anyone have any idea what could cause these differences?

**Comment by bsunanda** [comment]
@cvuosalo Please approve this PR

**Comment by bsunanda** [comment]
@cvuosalo This PR only modifies the print statements from out to LogVerbatim and use uniform token name "HCalGeom" rather than a mixture of "HCalGeom" and "HcalGeom". Please approve this PR

**Comment by cvuosalo** [comment]
Let's try the comparison test again.

**Comment by cvuosalo** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
The comparison differences have disappeared in the latest test. They were not related to this PR but may have briefly been in the IB used for previous tests.

**Comment by cvuosalo** [comment]
+1

**Comment by qliphy** [comment]
please test
just to double check possible differences

**Comment by qliphy** [comment]
+1
tests are stable now without essential differences.

**Pull request merged**